central/sensor/service/pipeline/complianceoperatorsuites/pipeline.go

```diff
@@ -0,0 +1,90 @@
+package complianceoperatorsuites
```

Comment:
For consistency and clarity I think we should name it `complianceoperatorsuitesv2`